test/Confluent.Kafka.IntegrationTests/Tests/Producer_ProduceAsync_Null_Task.cs

```diff
@@ -34,23 +35,22 @@ public static void Producer_ProduceAsync_Null_Task(string bootstrapServers, stri
         {
             var producerConfig = new Dictionary<string, object> 
             { 
-                { "bootstrap.servers", bootstrapServers },
-                { "api.version.request", true }
+                { "bootstrap.servers", bootstrapServers }
             };
 
             var drs = new List<Task<Message>>();
             using (var producer = new Producer(producerConfig))
             {
-                drs.Add(producer.ProduceAsync(partitionedTopic, null, 0, 0, null, 0, 0, 1, true));
-                drs.Add(producer.ProduceAsync(partitionedTopic, null, 0, 0, null, 0, 0, 1));
```

Comment:
dito.
Rule of the day: never remove tests (prior to removing the implementation they're testing), just add! :)

Reply:
i retained this as is but added a separate test function for obsolete methods.